.travis.yml

```diff
@@ -141,7 +141,7 @@ before_install:
     fi
   # Install go 1.8, required for cgo -srcdir flag to work
   - eval "$(gimme 1.8)"
-  - echo -e "[go]\n  root = ${GOROOT}" >> .buckconfig.local
+#  - echo -e "[go]\n  root = $(go env GOROOT)" >> .buckconfig.local
```

Comment:
why is this commented out?

Reply:
I was struggling to pass the integration test on Travis CI. GOROOT was set somewhere to `/usr/local/go`, cause `go list` command to throw an error `go: cannot find GOROOT directory: /usr/local/go`.

Do you know how GOROOT gets the value `/usr/local/go` in Travis CI? Running integration tests locally doesn't have the problem.

Reply:
@linzhp, afaik we have a default goroot in [test buck file](https://github.com/facebook/buck/blob/2d5e69e901e774a221997bed1580d0828e5a3561/test/com/facebook/buck/go/BUCK#L7):
```
"GOROOT": read_config("go", "root", "/usr/local/go"),
```

so I assume it's the one and as such it should be overridden by the
```
[go]
  root = <some other root>
```